docs/release.md

```diff
@@ -0,0 +1,77 @@
+A "FIWARE release" contains versioned releases of all Generic Enablers found
+within the [catalogue](https://www.fiware.org/developers/catalogue/). All
+components must work seamlessly with the current context broker and each other.
+
+The following timetable is proposed for a typical FIWARE release cycle, where it
+is assumed that releases will follow a monthly cadence.
+
+## Release Prerequisites
+
+Additional GE development for each release cycle should close by the beginning
+of the month - ideally this should mean that all code to be included in the
+FIWARE release has been:
+
+-   Written
+-   Tested
+-   Documented
+-   Labelled as a [SemVer release](GE_Requirements.md#releases) on **GitHub**
+
+Any proposed [new entries](GE_Checklist.md#new-generic-enablers) to the
+catalogue **MUST** have been presented to the TSC
+
+Any component
+[transitioning to a full member](GE_Checklist.md#transition-to-full-membership)
+of the catalogue **MUST** have completed all "Must" requirements and have been
+presented as such to the TSC
+
+## Release timetable
+
+#### Week 1
+
+The formal commencement of the release process will be announced during the
+**first** TSC meeting of the month
```

Comment:
too restrictive, I would remain silent

Reply:
Could be _TSC meeting of the month_ could be reworded - not sure how, what I want to implement is:

1. First TSC - announce the pre-release candidates
2. Second TSC close the list and wait for stragglers
3. Third TSC - announce the release

It doesn't literally have to be the first of the month. The point being that the process closes two weeks after announcement.